pipeline/inputs/forward.md

```diff
@@ -17,6 +17,9 @@ The plugin supports the following configuration parameters:
 | Buffer\_Chunk\_Size | By default the buffer to store the incoming Forward messages, do not allocate the maximum memory allowed, instead it allocate memory when is required. The rounds of allocations are set by _Buffer\_Chunk\_Size_. The value must be according to the [Unit Size ](../../administration/configuring-fluent-bit/unit-sizes.md)specification. | 1024000 |
 | Tag_Prefix          | Prefix incoming tag with the defined value.|  |
 | Tag                 | Override the tag of the forwarded events with the defined value.|  |
+| Shared\_Key         | Shared key for secure forward authentication. |  |
+| Self\_Hostname      | Hostname for secure forward authentication.   |  |
+| Secutiry.Users      | Specify username and password pairs for secure forward authenticaiton. | |
```

Comment:
```suggestion
| Security.Users      | Specify the username and password pairs for secure forward authentication. | |
```

Reply:
can you add an example of the Security.Users param?

Reply:
Sure. I will do.